src/sentry/static/sentry/app/components/activity/note/input.tsx

```diff
@@ -131,40 +114,42 @@ class NoteInput extends React.Component {
     this.setState({preview: true});
   };
 
-  handleSubmit = e => {
+  handleSubmit = (e: React.MouseEvent<HTMLFormElement>) => {
     e.preventDefault();
     this.submitForm();
   };
 
-  handleChange = e => {
+  handleChange = (e: MentionChangeEvent) => {
     this.setState({value: e.target.value});
 
     if (this.props.onChange) {
       this.props.onChange(e, {updating: !!this.props.modelId});
     }
   };
 
-  handleKeyDown = e => {
+  handleKeyDown = (
+    e: React.KeyboardEvent<HTMLTextAreaElement> | React.KeyboardEvent<HTMLInputElement>
+  ) => {
     // Auto submit the form on [meta] + Enter
     if (e.key === 'Enter' && (e.metaKey || e.ctrlKey)) {
       this.submitForm();
     }
   };
 
-  handleCancel = e => {
+  handleCancel = (e: React.MouseEvent<Element>) => {
     e.preventDefault();
     this.finish();
   };
 
-  handleAddMember = (id, display) => {
+  handleAddMember = (id: React.ReactText, display: string) => {
```

Comment:
I've seen that we usually use ReactText (number | string) but then we convert to string. shouldn't we only use the type string instead?  :D